### PR TITLE
update maintainers list in cabal file

### PR DESCRIPTION
**Pull request by andrewthad**

### CODE REVIEW

File: unordered-containers.cabal [hunk 1]
```diff
@@ -11,7 +11,7 @@ description:
 license:        BSD3
 license-file:   LICENSE
 author:         Johan Tibell
-maintainer:     johan.tibell@gmail.com
+maintainer:     johan.tibell@gmail.com, David.Feuer@gmail.com
```
Comment (sjakobi) [inline unordered-containers.cabal:14]: @emilypi and I also help with maintenance now, but @treeowl is the "primary maintainer" IMHO, so this addition seems right to me.

@tibbe Are you still an active maintainer, too?

Comment (tibbe) [reply unordered-containers.cabal:14]: Not anymore but I'm happy to answer questions about the design decisions involved in the library.

Comment (sjakobi) [reply unordered-containers.cabal:14]: Many thanks for being so responsive with the questions @tibbe! :)

I was briefly considering whether @emilypi and I should also be added, but I think that can wait until things have settled a bit. Do you agree @emilypi?

I also came up with "David Feuer and the unordered containers" which I think would be a great band name! ;)

Comment (emilypi) [reply unordered-containers.cabal:14]: I'm good with being added now or later. Maybe after a few legit PR's between us 
@sjakobi?

Comment (sjakobi) [reply unordered-containers.cabal:14]: > Maybe after a few legit PR's between us @sjakobi?

Sounds good to me!

**Pull request merged**